xmtp_mls/IDENTITY.md

```diff
@@ -54,6 +54,7 @@ XMTP installations consist of a long-lived Ed25519 key-pair (the 'installation k
    XMTP: Grant Messaging Access
 
    Current Time: <ISO 8601 date and time with local UTC offset>
```

Comment:
One thing I just realized, this leaks the user's timezone. Curious on other's thoughts here

Reply:
Good catch. Any harm in converting to UTC? Guess it makes it a little less human readable, but I can't imagine people are going to be reading these timestamps too closely

Reply:
Then we can't really ask users to double-check the timestamp, and we're trusting the app to furnish a valid timestamp. I suppose it's okay if a malicious app forges a fake timestamp here, as it's only used for identification? I can change it for now